frontend/src/actions/TracingActions.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionType, createAction, createStandardAction } from 'typesafe-actions';
+import { ActionKeys } from './ActionKeys';
+import { TracingInfo, JaegerTrace } from 'types/TracingInfo';
+
+// synchronous action creators
+export const TracingActions = {
+  setInfo: createStandardAction(ActionKeys.TRACING_SET_INFO)<TracingInfo | null>(),
+  setTrace: createAction(ActionKeys.TRACING_SET_TRACE, resolve => (trace?: JaegerTrace) =>
```

Comment:
I think this is still correct, right?  Because it is stilla JaegerTrace format that we retrieve?

Reply:
Yes, that is right